core/src/jvmMain/cpp/sqlite_bindings.cpp

```diff
@@ -0,0 +1,91 @@
+#include <jni.h>
```

Comment:
Mentioned in the PR note, this is copied from `androidMain`, would be good to share the file